### PR TITLE
Propagate secrets into release_artifacts workflow call

### PR DESCRIPTION
**Pull request by eatonphil**

### CODE REVIEW

**Review by batiati — DISMISSED** [review]
I'm just thinking out loud, we could standardize the secrets names/prefixes in a next opportunity, (e.g `NODE`, `GO`, `JAVA`, `DOTNET`)

**Comment by eatonphil** [comment]
Yeah good point.

**Comment by eatonphil** [comment]
Sorry had to fix a redundant call to the release artifacts step in pipeline.yml. Needs a reapproval when you can.

**Review by batiati — APPROVED** [review]

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -25,10 +25,6 @@ jobs:
     uses: ./.github/workflows/windows.yml
   macos:
     uses: ./.github/workflows/macos.yml
-  release_test:
```
Comment (eatonphil) [inline .github/workflows/pipeline.yml:28]: This was more necessary before when releases were once a week. But now this runs much of the whole steps twice on every run. When uploads should already be gated by the equivalent of when dry-run was false. So just dropping completely.

**Comment by eatonphil** [comment]
Thank you!

**Pull request merged**